### ISSUE TITLE
video crashed, is possible to recover?

### ISSUE THREAD

**Issue by Lyzoic-bit** [Genymobile/scrcpy#3172]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [e.g. Debian, Windows, macOS...]
 - scrcpy version: [e.g. 1.12.1]
 - installation method: [e.g. manual build, apt, snap, brew, Windows release...]
 - device model:
 - Android version: [e.g. 10]

**Describe the bug**

Hi, I've recorded long file but it got crashed and corrupted, is it possible to recover those videos anyhow?

**Comment by rom1v** [comment]
> Hi, I've recorded long file but it got crashed and corrupted, is it possible to recover those videos anyhow?

How did you record? (the exact command)
Which scrcpy version?

> it got crashed

Did scrcpy "crashed" ? What happened? Any error message?

**Comment by rom1v** [comment]
Could you share the video file?

**Comment by Lyzoic-bit** [comment]
It was mp4 video file, and not any error messages

**Comment by rom1v** [comment]
> How did you record? (the exact command)
> Which scrcpy version?
> Did scrcpy "crashed" ? (or did you interrupt the recording in some way)

**Comment by Lyzoic-bit** [comment]
there is code 0xc00d36c4

**Comment by rom1v** [comment]
?

**Comment by Lyzoic-bit** [comment]
The scrpy " crashed" while recording, and when I click the corrupted file it says 0xc00d36c4, it seems it got some frame-changes while recording or so(?), the wondershare said it can be fixed when I put sample video that had same environment as the recorded one (and it can be fixed there but obviously it wants me to buy the software) , anyway I don't know how this happened

**Comment by rom1v** [comment]
Does the file work with VLC?

**Comment by Lyzoic-bit** [comment]
not really, I tried but whether I'm bad at setting the attributes or needs on elsewhere

**Comment by kajeagentspi** [comment]
I was able to recover mine with [untrunc](https://github.com/anthwlock/untrunc).
